libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/ErrorResponse.java

```diff
@@ -90,8 +90,8 @@ public enum ErrorResponse {
     INVALID_JOURNEY_EVENT(1073, "Invalid journey event in input"),
     FAILED_TO_STORE_IDENTITY(1074, "Failed to store identity"),
     MITIGATION_ROUTE_CONFIG_NOT_FOUND(
-            1069, "No mitigation journey route event found in cimit config"),
```

Comment:
1069 isn't used above is the gap deliberate?

Reply:
Im seeing gaps inbetween a bunch of the error codes, not sure if its done for a reason or just errors have come and gone and therefore need to be unique going fowards for logging sake